content/guidance/inventory-code.md

```diff
@@ -13,4 +13,163 @@ sidenav: true
 sticky_sidenav: true
 ---
 
-Creating your enterprise code inventory
+# Creating your enterprise code inventory
+## Overview
+<p>
```

Comment:
<p> tags can be removed in all subsections in this page, not necessary but otherwise this shows up great in the page!